Prepare 1.0.1 release (#20)
jenkinsci/datadog-plugin: README.md

### Overview

The [jenkinsci/datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. 

> Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.

Reviewer: ```suggestion
Releases are merged to the [Jenkins-CI git repository for the Datadog-plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.
```

Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `Jenkinsfile` at the root folder of the source code.

A list of releases is available at [jenkinsci/datadog-plugin/releases][10].

> ### How to release

Reviewer: I think we should do a bullet list here:

```
To release a new Datadog-plugin version:

1. Change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 
2. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
3.  Clone to repository and checkout the `master` branch.
4. For repo admins only - Trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
If the command completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propagation time).

To release a new plugin version, change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 

> Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all the changes are listed accurately. 

Reviewer: ```suggestion
Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
```

### Overview

The [jenkinsci/datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. 
Releases are merged to the [Jenkins-CI git repository for the Datadog-plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.

Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `Jenkinsfile` at the root folder of the source code.

A list of releases is available at [jenkinsci/datadog-plugin/releases][10].

### How to release

To release a new Datadog-plugin version:

1. Change the project version in the [pom.xml][11] from `x.x.x-SNAPSHOT` to the updated version number you would like to see. 
2. Add an entry for the new release number to the [CHANGELOG.md][12] file, and ensure that all changes are listed accurately. 
3. Clone to repository and checkout the `master` branch.

> 4. For repository admins only - Trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.

Reviewer 2: ```suggestion
4. For repository admins only - trigger the release by running the following command `mvn -DlocalCheckout=true release:prepare release:perform`.
```

If the command completes successfully, then the newly updated plugin should be available from the Jenkins [Update Center][3] within ~4 hours (plus mirror propagation time).

## Issue Tracking

> Github's built-in issue tracking system is used to track all issues relating to this plugin: [jenkinsci/datadog-plugin/issues][13]. 

Reviewer 2: `s/Github/GitHub/`

However, given how Jenkins plugins are hosted, there may be issues that are posted to JIRA as well. You can check [this jenkins issue][14] for those issue postings.

> **Note**: [Unresolved issues on JIRA mentioning Datadog.][15].

Reviewer: ```suggestion
**Note**: [Unresolved issues on JIRA mentioning Datadog][15].
```

### Overview

The [jenkinsci/datadog-plugin][8] repository handles the most up-to-date changes made to the Datadog Plugin, as well as issue tickets revolving around that work. 
Releases are merged to the [Jenkins-CI git repo for our plugin][9], and represents the source used for plugin releases found in the [Update Center][3] in your Jenkins installation.

> Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `JenkinsFile` at the root folder of the source code.

Reviewer 2: ```suggestion
Every commit to the repository triggers the Jenkins Org CI pipeline defined in the `Jenkinsfile` at the root folder of the source code.
```
